src/modules/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import classNames from 'classnames/bind';
+
+import { QuickLinks } from '../QuickLinks';
+import { WhatsNew } from '../WhatsNew';
+import styles from './SideBar.module.css';
+
+const cn = classNames.bind(styles);
+
+export function SideBar() {
+  return (
+    <div className={cn('side-bar')}>
+      <WhatsNew />
```

Comment:
May be it should pass sidebar__item (for example) class to whatsnew to control their styles from this component, or we could wrap it to div with item className. Because we have to divide WhatsNew module from QuickLinks by margin, but those components doesn`t includes (or mustn`t if includes now) styles which changes their environment (margins for example) (looks for BEM https://en.bem.info/methodology/quick-start/)